src/modules/addlist.js

```diff
@@ -1,24 +1,82 @@
-import tasksToDo from './Array.js';
-
-export default class ContentList {
-  static displayToDoList = () => {
-    const toDoListContainer = document.querySelector(
-      '.to-do-list',
-    );
-    // sort the tasksToDo array by index
-    tasksToDo.sort((a, b) => a.index - b.index);
-    tasksToDo.forEach((task) => {
-      const listItem = document.createElement('div');
-      listItem.className = 'to-do-list-item';
-      listItem.innerHTML = `<li>
-   <input type="checkbox" class ="to-do-list" id="${task.index}" name="${task.description}" value="${task.description}"> ${task.description}
-  </li>
-  `;
-      const optionBtn = document.createElement('button');
-      optionBtn.className = 'option-btn';
-      optionBtn.innerHTML = '<i class="fa fa-ellipsis-v" aria-hidden="true"></i>';
-      listItem.appendChild(optionBtn);
-      toDoListContainer.appendChild(listItem);
-    });
-  };
+import clearCheckedItems from './delete.js';
+import editTask from './Update.js';
+import saveData from './saveData.js';
+
+const listItems = document.getElementById('list-container');
+const InputBox = document.getElementById('input-box');
+const clearBtn = document.getElementById('close-btn');
+const addTask = document.getElementById('addTask');
+
+function addTask1() {
```

Comment:
As mentioned by the previous reviewer and as stated in the project requirements, new tasks should have the property completed set to false and the property index set to the value of the new array length. This has been partially implemented as new tasks have only the `description` and `completed` properties without having the index property as seen in the screenshot below:

<img width="271" alt="image" src="https://github.com/AleWaweru/TODO-LIST/assets/37116322/a24a337c-77c5-47f6-9537-523f5bbebe6e">

Please make sure that new tasks added have all the required properties and adhere to the project requirements🙏

Reply:
hello, when you click the decrition item and edit, the index appears. please check that

Reply:
description*

Reply:
done